### PR TITLE
Review of Microverse Trials codes

### PR DESCRIPTION
**Pull request by Oscah01**
add microverse trials files

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        <div id="img-caption">A tribute to Elon Musk, The Richest Man Alive</div>
+      </div>
+      <div id="tribute-info">
+        <p>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Praesentium
+          assumenda error obcaecati odit quia qui asperiores dolorem, enim nemo
+          ab sapiente repellendus molestiae hic nam a eaque aperiam cum debitis.
+        </p>
+      </div>
+      <br>
```
Comment (Oscah01) [inline index.html:33]: try not to use br in spacing, instead, you can use padding and margin

File: index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+          alt=""
+          id="image"
+        />
```
Comment (Oscah01) [inline index.html:23]: id="image"
        />
"The image closing tag should be immediately after the code, it should be  id="image"/>

**Review by Oscah01 — COMMENTED** [review]
Check comments under the review

**Review by vechicin — APPROVED** [review]
# Hi @Oscah01 ✋, 

Your project is complete! 🎯 🧠 💯 
There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions 📂 

- [ ] Please keep in mind that reviews need to be as professional as possible in order to become more credible. Please make sure that you respect grammar rules (starting sentences with capitalized words, using proper indentation, etc.) for future reference.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**